tools/utils/download_blender.sh

```diff
@@ -117,9 +149,13 @@ if [ $# -ne 2 ]; then
 fi
 
 version=${1}
-current_dir=`pwd`
 output_dir=${2}
 
+if [ -z "${output_dir}" ]; then
```

Comment:
Do we need to check `version` as well?
If so, it would be nicer to check and output usage.

For example,

```bash
if [ #? -ne 2 ]; then
    echo "Usage: download_blender.sh <version> <output_dir>"
    exit 1
fi
```

Reply:
`${version}` is being checked in the following lines.

The only reason I am checking for `${output_dir}` so early is to avoid any root folders, as the script is executing `rm -r "${targetpath}"` whereas `local targetpath="${output_dir}/${target}"` which can result in `targetpath=/` and hence `rm -r /`.

Reply:
Understood.
But, we need to be fixed in future because this script requires two options.
So, early stopping is good way not to be unexpected behavior.